components/Notifications/Notification.styled.js

```diff
@@ -0,0 +1,15 @@
+import styled from "styled-components";
+
+export const NotificationContainer = styled.div`
+  position: absolute;
+  top: 43%;
+  left: 50%;
+  padding: 10px 20px;
+  transform: translate(-50%, -50%);
+  box-shadow: 0px 0px 10px rgba(0, 0, 0, 0.1);
+  padding: 1rem 2rem;
+  z-index: 10;
+  border-radius: 5px;
+  background-color: ${({ type }) =>
+    type === "add" ? "#DFF2BF" : type === "remove" ? "#FFBABA" : "black"};
+`;
```

Comment:
this might be hard to read for some devs, you could use another approach:
```js
background-color: black;

${({type}) => type === 'add' && css`
  background-color: #DFF2BF;
`}

${({type}) => type === 'remove' && css`
  background-color: #FFBABA;
`}
```
Depending on the type only one of the callback functions return styling, this pattern helps readers to distinguish your different stylings.